integration-tests/cli/csv-parser-price.it.js

```diff
@@ -119,6 +116,26 @@ describe('CSV and CLI Tests', () => {
         },
       )
     })
+
+    test('should log errors to stderr and log file', (done) => {
```

Comment:
Probably we change the word _log errors to stderr_. That's the final result or something, not really a log. Logging is somewhat different